### ISSUE TITLE
[Enhancement]: Making WooCommerce show products as if in a portfolio

### ISSUE THREAD

**Issue by paaljoachim** [woothemes/woocommerce#41085]
### Describe the solution you'd like

I have a client whom only wants to show the products with no prices included as well as no cart or payments.
Having a simple way to turn off prices, payments and everything else that is involved with selling a product would be helpful. As one could start basic with a kind of product portfolio page and when ready turn on the estore feature of prices, payments, cart etc.

### Describe alternatives you've considered

I have used CSS to hide prices from the store and individual product page.
Next is going through the WooCommerce settings to see if there is anything else that I should uncheck.
Hiding the WordPress backend left menu Analytics and Marketing tabs.

Bottom line is creating a vary basic simple setup for the usage of showing only a kind of portfolio and having a simple way to turn on additional functionality when needed.

### Additional context

Perhaps include it in the setup or atleast somewhere in the settings.
To where one can use WooCommerce as a portfolio hiding the estore functionalities.

**Comment by nerrad** [comment]
Hi Paal 👋🏻 

This is an interesting feature request but also mostly bespoke. It's not something we would be including in Woo core and would be better as an extension. Since we won't be actioning this issue I'm going to go ahead and close it.

**Comment by paaljoachim** [comment]
Hey Darren @nerrad 👋🏻

I would start most basic. In being able to hide Payments, Analytics and Marketing left sidebar tabs. As for various reasons these might just not be used. I say this because WooCommerce uses 5 left sidebar menu tabs/top level entries. I do not know of any other plugin that uses that much space in the left WP backend menu. 

Perhaps it goes back to having WooCommerce have more respect for the backend left sidebar menu. To not fill it up but to help the user clean up here if they would like so. I assume one can today use code to hide these areas but having a way in the advanced settings to hide these areas would be helpful.

Anyhow that is something to think about.

**Comment by nerrad** [comment]
Woo is rethinking its admin in parallel with the work on Phase 3 in WordPress. So IA is very much top of mind and will be incorporated into future work.

**Comment by paaljoachim** [comment]
That sounds great! I look forward to giving feedback when possible.
